infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -59,7 +59,7 @@
 
     public LogReplicationMetadataManager(CorfuRuntime rt, long topologyConfigId, String localClusterId) {
         this.runtime = rt;
-        this.corfuStore = new CorfuStore(runtime);
+        this.corfuStore = new CorfuStore(runtime, false);
```

Comment:
It will be good to add a comment here that LR simply replicates the streams and does not need it to be added to the transaction log